README.md

```diff
@@ -854,6 +855,120 @@ registration: {
 ```
 - **registration.click** - Function that is called when the registration button is clicked
 
+
+## Registration
+
+```javascript
+    var signIn = new OktaSignIn({
+      baseUrl: 'https://acme.okta.com',
+      clientId: '{{myClientId}}', // REQUIRED
+      registration: {
+        parseSchema: function(schema, onSuccess, onFailure) {
+           // handle parseSchema callback
+        },
+        preSubmit: function (postData, onSuccess, onFailure) {
+           // handle preSubmit callback
+        },
+        postSubmit: function (response, onSuccess, onFailure) {
+            // handle postsubmit callback
+        }
+      },
+      features: {
+        // Used to enable registration feature on the widget. https://github.com/okta/okta-signin-widget#feature-flags
+         registration: true // REQUIRED
+      }
+    });
+```
+
+Optional configuration:
+
+- **parseSchema:** Callback used to mold the JSON schema that comes back from Okta API.
+
+    ```javascript
+    // The callback function is passed 3 arguments: schema, onSuccess, onFailure
+    // 1) schema: json schema returned from the API.
+    // 2) onSuccess: success callback.
+    // 3) onFailure: failure callback. Note: accepts an errorObject that can be used to show form level or field level errors.
+
+    parseSchema: function (schema, onSuccess, onFailure) {
+      // This example will add an additional field to the registration form
+        schema.profileSchema.properties.address = {
+          'type': 'string',
+          'description': 'Street Address',
+          'default': 'Enter your street address',
+          'maxLength': 255
+        };
+        schema.profileSchema.fieldOrder.push('address');
```

Comment:
I'm a bit confused what is happening here. Is this notifying that changes were made to the `profileSchema` or is the `push` supposed to take an `address` object?

```javascript
var address = {
  ...
}
schema.profileSchema.fieldOrder.push(address);
```

Reply:
This example is showing how to use the parseSchema callback to modify the json schema. The assumption here is that the dev is familiar with json schema in general. schema.profileSchema will contain the list of fields returned from the API and all we are doing in the callback is creating a new `address` field and adding it to the list (in this case to the end).